flaskapp/view_functions/ivrflow.py

```diff
@@ -282,44 +292,47 @@ def search():
     it = 0
     str = ''
     for result in results:
-        #title=result.get('title')
-        res=result.get('snippet')
-        #name = result.get('displayLink')
-        str= str + f'{lst_num[it]} result: {res}".\n'
-        it=it+1
+        # title=result.get('title')
+        res = result.get('snippet')
+        # name = result.get('displayLink')
+        str = str + f'{lst_num[it]} result: {res}".\n'
+        it = it + 1
         if it == 3: break
```

Comment:
Instead of introducing and incementing the counter `it`, I would rewrite the loop as follows:
`for it, result in enumerate(results[:3]): ...`;  However, I don't see that `lst_num` is defined within the function scope (it is probably imported from somewhere... ); `3` is a magic number (bad), we need to define some parameter in the config file, e.g. `MAX_SEARCH_ITEMS = 3`.

Reply:
done